assets/sass/protocol/base/elements/_links.scss

```diff
@@ -8,8 +8,11 @@
     color: $link-color;
     text-decoration: underline;
 
+    &:visited {
+        color: $link-color-visited;
+    }
+
     &:hover,
-    &:focus,
```

Comment:
Declaring the colour change on focus was the root problem in https://github.com/mozilla/protocol/issues/842